src/components/Nav/_Nav.scss

```diff
@@ -30,4 +30,14 @@
     right: 0;
   }
 
+  // .tutorial::before {
```

Comment:
These lines are commented because they are syntax that I need to remember for a future implementation, _feel free to ignore_